sshkey_paramiko.r2py

```diff
@@ -94,9 +94,8 @@ class _sshkey_paramiko_BER(object):
   def __init__(self, content=''):
       self.content = content
       self.idx = 0
-
-  def decode(self):
-      return self.decode_next()
+  
+  # June 27, 2017: removed decode function, as it is not allowed by safe.py 
```

Comment:
Sounds good. (The Git metadata contains the date however, so there's no real need to add it to the comment.)